src/bin/purge_ttl.rs

```diff
@@ -132,21 +163,73 @@ fn main() -> Result<(), Box<dyn Error>> {
         {
             let _timer_batches = start_timer(&statsd, "purge_ttl.batches_duration");
             let mut req = prepare_request(&client, &session)?;
-            req.set_sql("DELETE FROM batches WHERE expiry < CURRENT_TIMESTAMP()".to_string());
-            let result = client.execute_sql(&req)?;
+            req.set_sql("SELECT (collection_id, fxa_kid, fxa_uid, batch_id) FROM batches WHERE expiry < CURRENT_TIMESTAMP()".to_string());
+            let result = SyncResultSet { result: client.execute_sql(&req)? };
+            let mut sql = "DELETE FROM batches WHERE (collection_id, fxa_kid, fxa_uid, batch_id) IN (".to_string();
```

Comment:
Tempted to suggest just leaving this as a `&str` and then doing the `format!` on demand on line 179 & 192, but whatever.